env/db/pg.go

```diff
@@ -60,4 +123,8 @@ func init() {
 			DBNumConns = int(toInt)
 		}
 	}
+
+	DBWriteURL = "postgresql://" + writeUserName + ":" + writePwd + "@" + writeHost + ":5432/" + writeDbName
+	DBReadURL = "postgresql://" + readUserName + ":" + readPwd + "@" + readHost + ":5432/" + readDbName
```

Comment:
This will force an override so DB_URL is never really used. Needs to set this inside the `else` logic when `DB_URL` is empty.

Reply:
oh, right